coldfront/core/project/migrations/0006_historicalproject_institution_project_institution.py

```diff
@@ -0,0 +1,26 @@
+# SPDX-FileCopyrightText: (C) ColdFront Authors
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# Generated by Django 4.2.11 on 2025-04-27 19:07
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("project", "0005_alter_historicalproject_options_and_more"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="historicalproject",
+            name="institution",
+            field=models.CharField(blank=True, default="None", max_length=10),
+        ),
+        migrations.AddField(
+            model_name="project",
+            name="institution",
+            field=models.CharField(blank=True, default="None", max_length=10),
```

Comment:
Looks like the migration just needs to be updated to reflect the change in length in the model. This should be `max_length=80`

Reply:
@co505 Looks good, thanks again for all the great work and best of luck with everything!